hnn_core/parallel_backends.py

```diff
@@ -56,12 +56,17 @@ def _gather_trial_data(sim_data, net, n_trials, postproc):
         net.cell_response._vsoma.append(spikedata[3])
         net.cell_response._isoma.append(spikedata[4])
 
+        N_pyr_x = net.params['N_pyr_x']
+        N_pyr_y = net.params['N_pyr_y']
+        dpls[-1]._baseline_renormalize(N_pyr_x, N_pyr_y)  # XXX cf. #270
+        dpls[-1]._convert_fAm_to_nAm()  # always applied, cf. #264
```

Comment:
this is different from the definition of a "raw dipole" in the GUI? I am fine with this but perhaps GUI users should weigh in.

@kohl-carmen @rythorpe any opinions?

Reply:
I'm not sure what a "raw dipole" is?